src/views/HubView.test.tsx

```diff
@@ -0,0 +1,73 @@
+import renderDashboardApp from "../tests/testUtils"
+import { wait } from "@testing-library/dom"
+import AxiosMockAdapter from "axios-mock-adapter"
+import { jinadInstance } from "../services/jinad"
+
+let mockJinadClient: AxiosMockAdapter
+const mockImages = [
+  {
+    "docker-name": "jinahub/pod.crafter.arraybytesreader",
+    version: "0.0.8",
+    "jina-version": "1.0.8",
+    "docker-command":
+      "docker pull jinahub/pod.crafter.arraybytesreader:0.0.8-1.0.8",
+    name: "ArrayBytesReader",
+    description:
+      "converts a byte stream into a numpy array and save to the Document.",
+    type: "pod",
+    kind: "crafter",
+    keywords: ["Some keywords to describe the executor", "separated by commas"],
+    platform: ["linux/amd64"],
+    license: "apache-2.0",
+    url: "https://jina.ai",
+    documentation: "https://github.com/jina-ai/jina-hub",
+    author: "Jina AI Dev-Team (dev-team@jina.ai)",
+    avatar: null,
+  },
+  {
+    "docker-name": "jinahub/pod.crafter.arraystringreader",
+    version: "0.0.8",
+    "jina-version": "1.0.8",
+    "docker-command":
+      "docker pull jinahub/pod.crafter.arraystringreader:0.0.8-1.0.8",
+    name: "ArrayStringReader",
+    description:
+      "convertsthe string of numbers into a numpy array and save to the Document.",
+    type: "pod",
+    kind: "crafter",
+    keywords: ["Some keywords to describe the executor", "separated by commas"],
+    platform: ["linux/amd64"],
+    license: "apache-2.0",
+    url: "https://jina.ai",
+    documentation: "https://github.com/jina-ai/jina-hub",
+    author: "Jina AI Dev-Team (dev-team@jina.ai)",
+    avatar: null,
+  },
+]
+
+describe("HubView integration test", () => {
+  beforeAll(() => {
+    mockJinadClient = new AxiosMockAdapter(jinadInstance)
+  })
+
+  beforeEach(() => {
+    mockJinadClient.onGet("/images").reply(200, mockImages)
```

Comment:
jinaD is the wrong client and has nothing to do with the hub.
You can check the documentation for information on jinaD
https://docs.jina.ai/chapters/remote/jinad

We also do integration tests with Cypress. 
https://www.cypress.io/
You can check out the cypress folder for examples

Reply:
thanks. Will check it and fix it over weekend. 👍

Reply:
yup. updated. almost all cypress test cases seems good. It just didn't had anything to cover `/home` so added that. 👍

Reply:
and updated `ImageCard` component to cover author name display case in cypress test suite.